exec/cnex/cnex.c

```diff
@@ -870,9 +911,17 @@ void exec_INA(TExecutor *self)
     push(self->stack, cell_fromBoolean(v));
 }
 
-void exec_IND()
+void exec_IND(TExecutor *self)
 {
-    fatal_error("exec_IND not implemented");
+    self->ip++;
+    Dictionary *dictionary = top(self->stack)->dictionary;
+    TString *key = peek(self->stack, 1)->string;
```

Comment:
Does this need to be freed in this function? It's unclear to me how this memory is managed.